middleware/src/index.js

```diff
@@ -96,8 +96,8 @@
         }
 
         payload = _maskSuspiciousAttributes(payload);
-        
-        payload = {...payloadglobalAttributes, payload}
+        payload = Object.assign({}, JSON.parse(payload), globalAttributes);
```

Comment:
why this parse here?

Reply:
because _maskSuspiciousAttributes returns String type

Reply:
then why don't we move the globalAttributes before calling it?